<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicializador_register</class>
 <widget class="QWidget" name="inicializador_register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Password</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Clowns_Password.ico</normaloff>../img/Clowns_Password.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="carregando_dados_register">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_novo_user_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_novo_password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_email">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Fixedsys&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_voltar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/seta.png</normaloff>../img/seta.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/Clowns_Password_png.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
